0-ad-history-mod-package/simulation/templates/units/ptol/siege_polybolos_packed.xml

```diff
@@ -19,13 +19,13 @@
     <SpecificName>Polybolos</SpecificName>
     <Icon>units/hele/siege_oxybeles_packed.png</Icon>
     <History>
-        The polybolos was a bolt-shooting device invented by Dionysius of Alexandria in the 3rd century B.C. Because it could fire automatically without manual reloading, it is often called "the first machine gun". It is an impressive piece of engineering and a remarkable achievement of the Ancient Greeks.
+        The polybolos was a bolt-shooting device invented by Dionysius of Alexandria in the 3rd century BC Because it could fire automatically without manual reloading, it is often called "the first machine gun". It is an impressive piece of engineering and a remarkable achievement of the Ancient Greeks.
```

Comment:
```suggestion
        The polybolos was a bolt-shooting device invented by Dionysius of Alexandria in the 3rd century BC. Because it could fire automatically without manual reloading, it is often called "the first machine gun". It is an impressive piece of engineering and a remarkable achievement of the Ancient Greeks.
```